python/dgl/sparse/sparse_matrix.py

```diff
@@ -91,43 +91,86 @@ def col(self) -> torch.Tensor:
         """
         return self.coo()[1]
 
-    def coo(self) -> Tuple[torch.Tensor, ...]:
-        """Returns the coordinate (COO) representation of the sparse matrix.
+    def coo(self) -> Tuple[torch.Tensor, torch.Tensor]:
+        """Returns the coordinate list (COO) representation of the sparse
+        matrix. See `COO in Wikipedia <https://en.wikipedia.org/wiki/
```

Comment:
The short description should be one (short) sentence so it can be rendered nicely. Put the "See ..." in a new line. E.g.,

```
"""Returns the coordinate list (COO) representation of the sparse matrix.

See `COO in Wikipedia <https://en.wikipedia.org/wiki/Sparse_matrix#Coordinate_list_(COO)>`_.

...
```